bin/cylc-report-timings

```diff
@@ -323,7 +356,9 @@ class HTMLTimingSummary(TimingSummary):
         import matplotlib.pyplot as plt
         buf.write('<div class="timing" id=%s>' % category)
         buf.write('<h2>%s</h2>\n' % (category.replace('_', ' ').title()))
-        ax = df_reshape.xs(category, level=3).plot(kind='box', vert=False)
+        ax = df_reshape\
+            .xs(category, level='timing_category')\
+            .plot(kind='box', vert=False)
```

Comment:
No need to use line continuation markers here - you can just break lines inside parentheses.